### PR TITLE
Feature: implement tests for zcnbridge

### LINKED ISSUE #622: Feature: implement tests for zcnbridge

### PR DESCRIPTION
**Pull request by YarikRevich**
# Changes
* Adds tests for DEX bridge methods. It's important to mention, that its main purpose it to test configurations used by DEX bridge methods, not the logic of ABIs.
* Fixes mock errors.

# Associated PRs
* https://github.com/0chain/zwalletcli/pull/298

### CODE REVIEW

File: zcnbridge/config.go [hunk 1]
```diff
@@ -24,43 +29,66 @@ type BridgeSDKConfig struct {
 }
 
 type BridgeClient struct {
+	KeyStore
+	transaction.TransactionProvider
+	EthereumClient
```
Comment (peterlimg) [inline zcnbridge/config.go]: We'd better keep the members private

**Review by peterlimg — COMMENTED** [review]
Please also fix the build errors in both zwalletcli and zboxcli, the changes like the transaction.Transaction interface will break the builds.

**Comment by YarikRevich** [comment]
@peterlimg Only zwallet requires changes and the required PR is prepared and attached in the description

File: zcnbridge/config.go (hunk 1 above)
Comment (YarikRevich) [reply zcnbridge/config.go]: Done

**Review by peterlimg — APPROVED** [review]
LGTM

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  |   |
| gosdk  | feature/zcnbridge-tests  |
| zboxcli  |   |
| zwalletcli  |   |
| 0chain  |   |
| blobber  |   |
| authorizer  |   |
| 0box  |   |
| 0dns  |   |

**Pull request merged**